bin/git-mv-remote

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+old=$1
+new=$2
+
+test -z $old && echo "old remote name required." 1>&2 && exit 1
+test -z $new && echo "new remote name required." 1>&2 && exit 1
+
+git remote remove $new || true
```

Comment:
Better to check if the `$new` remote exists before removing it, so that we won't see error message like `fatal: ....`.

Reply:
I just suppressed its output because I was feeling lazy, hope that's ok

Reply:
@timfeirg 
Hiding the error message is not a good idea...
Maybe you could use `git config --get 'remote.$name.fetch'` to check if a remote exists.

Reply:
fixed as suggested